src/docfx/lib/HtmlUtility.cs

```diff
@@ -15,6 +16,8 @@ internal static class HtmlUtility
         private static readonly Func<HtmlAttribute, int> s_getValueStartIndex =
             ReflectionUtility.CreateInstanceFieldGetter<HtmlAttribute, int>("_valuestartindex");
 
+        private static readonly Regex s_styleCellAlignRegEx = new Regex("^text-align: (?:(?:left)|(?:center)|(?:right));$", RegexOptions.Compiled);
```

Comment:
Looks like this regex means it must be exactly `text-align: left;` or `text-align: right;` or `text-align: center;`, so may be a `Contains` is enough.

On the other hand, `text-align:right` is also a valid style, but I don't think we need to go that far.

Reply:
`Contains` or `Equals`?

Reply:
I mean `Array.Contains`, you have an array of `['text-align: left', 'text-align: right', etc]`